### PR TITLE
Task4667 vuetify parallax docs

### PR DESCRIPTION
**Pull request by Sarin2023**

### CODE REVIEW

File: source/includes/documentation/_vue_js_elements.md [hunk 1]
```diff
@@ -2693,12 +2693,27 @@ For examples of usage see: [JDI vuetify page tests for Skeleton Loader](https://
```
Comment (MayaElf) [inline source/includes/documentation/_vue_js_elements.md:2693]: 1) It's good to have a phrase 'Parallax is located in the following class:'
2) Please, add this test also 'contentParallaxTests()'

File: source/includes/documentation/_vue_js_elements.md [hunk 2]
```diff
@@ -2693,12 +2693,27 @@ For examples of usage see: [JDI vuetify page tests for Skeleton Loader](https://
...
+     </div>
+       <div class="v-parallax__content"></div>
+    </div>
+</div>
+
+```
+
+v-parallax element has following methods:
+
 |Method | Description | Return Type
```
Comment (MayaElf) [inline source/includes/documentation/_vue_js_elements.md:2711]: Please,
1) add framework version
2) in a documentation near highlighted comment please also emphasize a Selenium locator findby 
image

File: source/includes/documentation/_vue_js_elements.md [hunk 3]
```diff
@@ -2693,12 +2693,27 @@ For examples of usage see: [JDI vuetify page tests for Skeleton Loader](https://
...
 --- | --- | ---
-**height()** | Returns the value of the parallax container height attribute | String
+**heightPx()** | Returns the value of the parallax container height attribute | int
 **hasContent()** | Checks if the parallax container has any elements within | boolean
 **image()** | Returns the background image of the parallax container (return type is JDI Light HTML Image) | Image
-**is()/has()** | Returns Assert class | ParallaxAssert
+**is()** | Returns Assert class | ParallaxAssert
 
 For examples of usage see: [JDI vuetify page tests for Parallax](https://github.com/jdi-testing/jdi-light/blob/vuetify-develop/jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/composite/ParallaxTests.java).
```
Comment (MayaElf) [inline source/includes/documentation/_vue_js_elements.md:2719]: You may find good example of 'documentation PR' here :) #1328

File: source/includes/documentation/_vue_js_elements.md [hunk 4]
```diff
@@ -2693,12 +2693,27 @@ For examples of usage see: [JDI vuetify page tests for Skeleton Loader](https://
...
+v-parallax element has following methods:
+
 |Method | Description | Return Type
 --- | --- | ---
-**height()** | Returns the value of the parallax container height attribute | String
+**heightPx()** | Returns the value of the parallax container height attribute | int
 **hasContent()** | Checks if the parallax container has any elements within | boolean
 **image()** | Returns the background image of the parallax container (return type is JDI Light HTML Image) | Image
-**is()/has()** | Returns Assert class | ParallaxAssert
+**is()** | Returns Assert class | ParallaxAssert
```
Comment (MayaElf) [inline source/includes/documentation/_vue_js_elements.md:2716]: Please, add other methods from ParallaxAssert :)

**Review by MayaElf — CHANGES_REQUESTED** [review]

File: source/includes/documentation/_vue_js_elements.md (hunk 1 above)
Comment (Sarin2023) [reply source/includes/documentation/_vue_js_elements.md:2708]: Modified as per the suggestions. Please review.

**Review by MayaElf — APPROVED** [review]

**Review by vklonin — APPROVED** [review]

**Pull request merged**